pkg/solver/server.go

```diff
@@ -388,6 +394,12 @@ func (server *solverServer) addResourceOffer(resourceOffer data.ResourceOffer, r
 		}
 	}
 
+	versionHeader, _ := http.GetVersionFromHeaders(req)
+	minVersion, ok := server.versionConfig.IsSupported(versionHeader)
+	if !ok {
+		return nil, fmt.Errorf("Please update to minimum supported version %s or newer: https://github.com/Lilypad-Tech/lilypad/releases", minVersion)
```

Comment:
Same comment here around adding a log statement

Reply:
Added in db9caa2b0b0609898642732b33941014cba8b448